src/hooks/users/useRecentLocationUsers.ts

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from "react";
+
+import { normalizeTimestampToMilliseconds } from "utils/time";
+
+import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
+
+import { useWorldUsers } from "./useWorldUsers";
+
+/**
+ * @description this hook's filtering world users based on their @lastSeenIn location
+ *
+ * @param locationName is a key for `lastSeenIn` firestore field in user's object
+ *
+ * @example useRecentLocationUsers(venue.name)
+ * @example useRecentLocationUsers(`${venue.name}/${roomTitle}`)
+ */
+export const useRecentLocationUsers = (locationName?: string) => {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.